### ISSUE TITLE
help file cocofilter

### ISSUE THREAD

**Issue by AlexPietrow** [mdruett/COCOPLOT#3]
I found a problem with the IDL documentation of the cocofilter function. 

The user can put in a wavelength array (that is not always equidistant) or just say how many points the array has. For single and band the user should give r,g,b in array element numbers and not in wavelength. And in normal it should always be wavelength and not array element number. Maybe even wavelength is confusing since its not always wavelength. 

Anyway, if wavelenths is an 21element array centered around 8542, then the user should not put in 8542.2 as r for single or band. But this is not clear from the documentation atm. 
```
;                 'single' : single wavelength points
;                            uses keywords r=r, g, b with integers
;                            specify which wavength points wanted 
;                            for each filter
```

**Comment by grviss** [comment]
Indeed, that's unclear now. I can update the program info to clarify that the `r`, `g` and `b` keywords take wavelength/time indices, rather than wavelength/time values.

**Comment by grviss** [comment]
Fixed in update #4
